atlas/templates/settings.php

```diff
@@ -44,6 +44,12 @@
 {% endif %}
 $conf['googleanalytics_account'] = 'UA-25752450-1';
 
+{% if google_tag_client_container_id %}
```

Comment:
Should be able to combine this and the nested conditional into a single statement with `and`